README.md

```diff
@@ -1 +1,36 @@
-# ARK-iOS
\ No newline at end of file
+# ARK-iOS
+
+## System Requirements
+* Ruby 2.6.5
+* Bundler 2.3.19
+
+## Setup(Mac)
+- Install ruby & bundler
+  ```sh
+  brew install ruby
+  bundle install
```

Comment:
Maybe should change to `gem install bundler`

Reply:
Good catch!!! 🙏